docs/topics/js/javascript-dce.md

```diff
@@ -1,5 +1,11 @@
 [//]: # (title: Kotlin/JS dead code elimination)
 
+> The DCE tool is deprecated. The DCE tool was designed for the legacy JS backend, which is now obsolete. The current 
+> [JS IR backend](#dce-and-javascript-ir-compiler) supports dead code elimination out of the box, and the @JsExport feature allows specifying which Kotlin 
+> functions and classes to retain during tree shaking.
```

Comment:
Using "tree shaking" here might be confusing. Although the term is very similar to DCE, it usually refers to JS [tooling](https://webpack.js.org/guides/tree-shaking/) and DCE to refer to Kotlin tooling, which might actually remove the `@JsExport`ed declarations if they don't end up being used by the JS code.

More detail. The difference between DCE and tree shaking is that Kotlin's DCE handles code produced by Kotlin and JS "tree shaking" ususally handles the whole bundle. Those two are not interchangeable and usually work together - DCE help remove Kotlin bloat, and tree shaking handles the whole bundle - both hand-written, and generated. It is less precise with Kotlin code but it can shave off some extra code by taking into account which exported declarations didn't end up being used by the "manual" JS code.

Reply:
Thanks for the explanation. I'll change tree shaking to DCE.